### ISSUE TITLE
snekfetch 3.4.2 POST not showing request body in Express & bodyparser

### ISSUE THREAD

**Issue by KararTY** [devsnek/snekfetch#16]
Using snekfetch post like this,
```js
snek.post(url + '/login/app').send({username: 'username', password: 'password'})
  .then(res => {
    console.log(res.body)
  })
```
in version **3.4.2** of snekfetch will give an empty body in express with the body parser module.
I've checked res.request in the snekfetch res object to make sure that the proper body was interpreted and it was.

But, using an older version of snekfetch, more specifically the version **3.2.9**, the above will actually function and the body will correctly show.

Any clues to why this might be happening with the latest version of snekfetch? I've reverted back to version 3.2.9 for now.

**Comment by devsnek** [comment]
can you specify if you're in browser or node
on latest release:
in node i have
![](https://gc.gy/蓮.png)
and in browser i have
![](https://gc.gy/◄ɉ.png)
i should also mention that i just made a change to browser which fixed some headers not being set (see 73724709c7bcb732165f12503066ba86cd922198)

**Comment by KararTY** [comment]
Using it in an Electron window, I'm assuming snekfetch automatically assumes it's a browser due to this,
```js
const browser = typeof window !== 'undefined';
```
in index.js, even though snekfetch can use node functionalities within an electron window.

Thank you though, it was fixed with that change.
